src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -150,6 +166,13 @@ private ReadFilterLibrary(){ /*no instance*/ }
         @Override public boolean test(final GATKRead read){
             return ! read.isDuplicate();}}
 
+    /** Keep only paired reads that are not properly paired (0x1 and !0x2). Removes single ended reads. */
```

Comment:
You might want to be explicit here and say "that are not marked as properly paired" -- just to highlight that they were marked as improperly paired by the aligner or another tool that manipulated the bam file, as opposed to logic implemented in this filter (this can be a point of confusion in my experience).

Reply:
Done